docs/concepts/configuration/pod-with-pod-affinity.yaml

```diff
@@ -5,14 +5,16 @@ metadata:
 spec:
   affinity:
     podAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-      - labelSelector:
-          matchExpressions:
-          - key: security
-            operator: In
-            values:
-            - S1
-        topologyKey: failure-domain.beta.kubernetes.io/zone
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: security
+              operator: In
+              values:
+              - S1
+          topologyKey: failure-domain.beta.kubernetes.io/zone
     podAntiAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
It would be good if you changed `podAntiAffinity` to `preferredDuringSchedulingIgnoredDuringExecution` and keep podAffinity set to requiredDuringSchedulingIgnoredDuringExecution in this file to ensure we are not implying that podAffinity should always be preferredDuringSchedulingIgnoredDuringExecution and podAntiAffinity should always be requiredDuringSchedulingIgnoredDuringExecution.